tolerance-policies/apps/monitor/src/test/java/com/redhat/training/NativeInfoResourceIT.java

```diff
@@ -1,7 +1,6 @@
 package com.redhat.training;
 
-import io.quarkus.test.junit.NativeImageTest;
-
+import io.quarkus.test.junit5.NativeImageTest
 @NativeImageTest
```

Comment:
`@NativeImageTest` is not available anymore.
Use `@QuarkusIntegrationTest`

Reply:
Also, apply this same fix to the solutions folder.

Reply:
Done